### ISSUE TITLE
Скрытая запись...

### ISSUE THREAD

**Issue by cisco-kisko** [rdp/screen-capture-recorder-to-video-windows-free#118]
Здравствуйте.
Я бы хотел узнать, у вас. 
Как я могу запустить вашу программу в скрытом режиме? При этом чтобы она автоматически начала запись видео. 

Я не очень опытен в програмировании. Если вы мне поможете, я буду вам очень благодарен!

**Comment by rdp** [comment]
like from the webcam?

**Comment by cisco-kisko** [comment]
Нет, мне нужно ее запустить скрыто, могли бы вы предоставить какой либо batch file?
Чтобы я мог ее запустить скрыто от моих подчиненных.

**Comment by rdp** [comment]
hidden capture the...screen? or webcam?  If you can install the software on
there then you can start ffmpeg.exe itself "manually" to start a
recording...set it to start when user login, record to an mpg file so it
doesn't matter if recording interrutped when they logoff, etc.

**Comment by rdp** [comment]
I'm not sure on the details there but you'd think it would be
possible...good luck!  Just manually call ffmpeg somehow, either save to
local file or send to a server maybe?
